src/mockBatchCreator.ts

```diff
@@ -11,6 +11,20 @@ const mockFunction = function() {
     return null;
 };
 export default class _BatchValidator {
+    private configOverride?: {omitBatchTimestamp?: boolean};
+    private storeOverride?: {batchTimestampUnixtimeMsOverride?: number}
```

Comment:
We're trying to get into the habit of being more explicit and intentional with typing of config settings and overrides, so I would ask if you can add SDKConfig and Store as the type here.

Reply:
Makes sense. I actually did that at first, but I didn't want to give the impression that _any_ field could be overridden via this config, as I only implemented the overrides for these two fields.

I changed it to use `Partial<Pick<SDKConfig, 'omitBatchTimestamp'>>` and `Partial<Pick<IStore, 'batchTimestampUnixtimeMsOverride'>>`, which uses the derived type but ensures that only the implemented fields can be overridden.

Lmk what you think.